pkgs/applications/video/showmethekey/default.nix

```diff
@@ -0,0 +1,53 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, glib
+, meson
+, ninja
+, libevdev
+, json-glib
+, cairo
+, pango
+, libinput
+, gtk4
+, wrapGAppsHook
+, libxkbcommon
+, pkg-config
+}:
+stdenv.mkDerivation rec {
+  pname = "showmethekey";
+  version = "1.7.3";
+
+  src = fetchFromGitHub {
+    owner = "AlynxZhou";
+    repo = "showmethekey";
+    rev = "v${version}";
+    sha256 = "sha256-hq4X4dG25YauMjsNXC6Flco9pEpVj3EM2JiFWbRrPaA=";
+  };
+
+  nativeBuildInputs = [
+    glib
+    meson
+    ninja
+    cairo
+    pango
+    json-glib
+    pkg-config
+    libevdev
+    libinput
+    libxkbcommon
```

Comment:
Maybe some of these does not belong to nativeBuildInputs? :thinking:

Reply:
Yup, looks like it. I missed those, sorry :/

Reply:
glib and pango seems like not. sorry i just keep dependencies write by author but not check its useful. Should i remove it, or wait next version?

Reply:
What we're saying is that some belong in `buildInputs`. `nativeBuildInputs` is for things like build systems and `pkg-config` -- tools that have to execute during the build; this is to aid cross-compilation. See [this section](https://nixos.org/manual/nixpkgs/unstable/#sec-cross-packaging) of the manual for more information.

Reply:
So `cairo` and `pango` should belong to `buildInputs`?

Reply:
So `cairo` and `pango` should belong to `buildInputs`?